.github/ISSUE_TEMPLATE/test_case.md

```
@@ -1,26 +1,26 @@
-name: Тест-кейс
-about: Шаблон для создания нового тест-кейса
+name: Test Case
+about: Template for creating a new test case
 title: '[Test Case]: '
 labels: ''
 assignees: ''
 ---
 
-### Описание
-*Кратко опишите, какую функциональность проверяем.*
+### Description
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)